chameleonultragui/lib/gui/flashing.dart

```diff
@@ -32,14 +33,28 @@ class FlashingPage extends StatelessWidget {
               ),
               const SizedBox(height: 20),
               Text(
-                appState.easterEgg
+                appState.onWeb
+                    ? 'Select the firmware zip file to start flasing'
+                    : appState.easterEgg
                     ? 'Your ${appState.connector.deviceName} is flashing'
                     : 'Installing firmware on your ${appState.connector.deviceName}',
                 style:
                     const TextStyle(fontSize: 24, fontWeight: FontWeight.bold),
                 textAlign: TextAlign.center,
               ),
               const SizedBox(height: 10),
+              TextButton(
```

Comment:
Flashing page is only for displaying flashing, all work should be done with underlying methods (flashFirmwareZip) and etc. No calls should be in flashing page

Reply:
Yeah, you are right ofc. See other comment, have resolved this by adding a separate card for DFU devices on the connect page